### DISCUSSION TITLE
Patches removed in the refactor

### DISCUSSION THREAD

**Discussion by Francesco146** [anddea/revanced-patches#491] [General]
after the refactor made by merging inotia00's code, some patches were removed. (5b4dca0)

for now, me and @anddea are at this point:

* disable shorts like animation 
* force hide shorts dim
* speed whitelist
* sponsorblock whitelist 
* hide player buttons background inside settings
* hide animated button background

do you know any other patches that were removed?

**Comment by anddea** [comment]
Hide player buttons background inside settings.

**Reply by Francesco146** [reply]
https://github.com/anddea/revanced-patches/blob/main/src/main/kotlin/app/revanced/patches/youtube/player/playerbuttonbg/HidePlayerButtonBackgroundPatch.kt

this one, right?

**Reply by anddea** [reply]
Yes.
